decidim-core/app/helpers/decidim/traceability_helper.rb

```diff
@@ -22,7 +22,7 @@ def render_resource_last_editor(resource)
     #
     # Returns an HTML-safe String representing the HTML to render the author.
     def render_resource_editor(version)
-      cell "decidim/version_author", present(Decidim.traceability.version_editor(version)), context_actions: [:date], layout: :compact, has_actions: true, from: version
+      cell "decidim/redesigned_version_author", present(Decidim.traceability.version_editor(version)), context_actions: [:date], layout: :compact, has_actions: true, from: version
```

Comment:
Is this part of the revert?

Reply:
Not quite, as commented in the description I've included an additional change to use redesigned versions of cells if the `redesigned_` particle is explicitly used in the code. In this way we can remove the skip applied when redesign is not enabled in some cases. I have done this in this PR because the drawer functionality was used both in diff and meetings views and after reverting I found that some examples could be recovered if redesigned cells remained in use regardless the redesign global feature setting